src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -802,19 +804,17 @@ def send_raw_request(cli_ctx, method, url, headers=None, uri_parameters=None,  #
         else:
             logger.warning("Can't derive appropriate Azure AD resource from --url to acquire an access token. "
                            "If access token is required, use --resource to specify the resource")
-    try:
-        # https://requests.readthedocs.io/en/latest/user/advanced/#prepared-requests
-        s = Session()
-        req = Request(method=method, url=url, headers=headers, params=uri_parameters, data=body)
-        prepped = s.prepare_request(req)
-
-        # Merge environment settings into session
-        settings = s.merge_environment_settings(prepped.url, {}, None, not should_disable_connection_verify(), None)
-        _log_request(prepped)
-        r = s.send(prepped, **settings)
-        _log_response(r)
-    except Exception as ex:  # pylint: disable=broad-except
-        raise CLIError(ex)
```

Comment:
May I ask why the CLIError is not wrapped here any more?

Reply:
If we wrap the exception with `CLIError`, the exception type will be lost and 
https://github.com/Azure/azure-cli/blob/a0f8c8ecbec012b07c00441639f5866043b3c218/src/azure-cli-core/azure/cli/core/util.py#L82
won't be hit anymore.

We are leaving the error handing to `handle_exception` instead.

Reply:
Got it, thx~